app/models/person.rb

```diff
@@ -84,9 +92,10 @@ def target_legislators(json: false, count: Ivr::Call::MAXIMUM_CONNECTIONS)
   private
 
   def update_nation_builder
-    relevant_fields = changed & ['email', 'phone']
-    if relevant_fields.any?
+    relevant_fields = changed & ['email', 'phone', 'first_name', 'last_name']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.